doc/data/messages/b/bare-name-capture-pattern/bad.py

```diff
@@ -0,0 +1,13 @@
+red = 0
+green = 1
+blue = 2
+
+
+color = blue
+match color:
+    case red:  # [bare-name-capture-pattern]
+        print("I see red!")
+    case green:  # [bare-name-capture-pattern]
+        print("Grass is green")
+    case blue:
+        print("I'm feeling the blues :(")
```

Comment:
```suggestion
def func(color):
    match color:
        case red:  # [bare-name-capture-pattern]
            print("I see red!")
        case green:  # [bare-name-capture-pattern]
            print("Grass is green")
        case blue:
            print("I'm feeling the blues :(")
```
Wrapping the match statement in a function might help make it more obvious that color is a variable.